src/frontend/pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[build-system]
+# needs a version that supports PEPs 517, 518, 621, 660
+# https://setuptools.pypa.io/en/latest/history.html#v61-0-0
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "CellpPofiler"
+version = "5.0.0.dev1"
+authors = [
+    {name = "Anne Carpenter"},
+    {name = "Thouis (Ray) Jones", email = "thouis@gmail.com"},
+    {name = "Lee Kamentsky", email = "leek@broadinstitute.org"},
+    {name = "Vebjorn Ljosa", email = "vebjorn@ljosa.com"},
+    {name = "David Logan", email = "dlogan@broadinstitute.org"},
+    {name = "Mark Bray", email = "mbray@broadinstitute.org"},
+    {name = "Madison Swain-Bowden", email = "bowdenm@spu.edu"},
+    {name = "Allen Goodman", email = "allen.goodman@icloud.com"},
+    {name = "Claire McQuinn", email = "mcquin@users.noreply.github.com"},
+    {name = "Beth Cimini", email = "bcimini@broadinstitute.org"},
+    {name = "David Stirling", email = "dstirling@glencoesoftware.com"},
+    {name = "Alice Lucas", email = "alicelucas93@gmail.com"},
+    {name = "Nodar Gogoberidze", email = "ngogober@broadinstitute.org"},
+    {name = "Callum Tromans-Coia", email = "callumtc.jpg@gmail.com"},
+]
+maintainers = [
+    {name = "Nodar Gogoberidze", email = "ngogober@broadinstitute.org"},
+]
+description = """
+CellProfiler is a free open-source software designed \
+to enable biologists without training in computer vision \
+or programming to quantitatively measure phenotypes from \
+thousands of images automatically."""
+readme = "README.md"
+# we don't place upper bound pin for python:
+# https://iscinumpy.dev/post/bound-version-constraints/#pinning-the-python-version-is-special
+requires-python = ">=3.8"
+# can't reach outside
+# license = {file = "../../LICENSE"}
+license = {text = "BSD-3-Clause"}
+keywords = ["computer vision", "image analysis", "biology", "cell", "cellprofiler"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    "Topic :: Scientific/Engineering :: Image Processing",
+    "Topic :: Scientific/Engineering",
+]
+dependencies = [
+    "cellprofiler-library==5.0.0",
```

Comment:
For this and below, if we really want to keep everything in sync, can we pull the version directly?

Reply:
Not until the build system refactor, I don't think. We can dynamically version the package that a particular `pyproject.toml` refers to (see latest changes, and use of `setuptools-scm`), but we can't dynamically link a dependencies version.